src/main/scala/lantern/ad_lms_vector.scala

```diff
@@ -287,6 +296,9 @@ trait TensorExp extends Dsl with Diff {
     * cuBLAS tensor operation backend. WIP.
     */
   class BackendCublas extends Backend {
+    override def setup(): Unit = unchecked[Unit]("cublasHandle_t handle;\nCUBLAS_CALL(cublasCreate(&handle));")
```

Comment:
This `unchecked` call is not useful by itself because it doesn't actually get code generated. (Perhaps it's removed by DCE since it has no users.)
Do you know how I can cause it to be generated? @TiarkRompf @feiwang3311 

Please hold off on merging until this is fixed.

Reply:
Fixed in https://github.com/feiwang3311/Lantern/pull/20/commits/c00698c75f9bb2715d1c214bb8715f9d6294dafe.